cms/migrations/0043_setup_certificate_index_page.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 2.2.4 on 2019-09-18 07:20
+
+from django.db import migrations
+from wagtail.core.models import Page
+
+from cms.constants import CERTIFICATE_INDEX_SLUG
+
+
+def get_home_page(apps):
+    """
+    Importing the Site model from the registry means if we access the root page from this
+    model we will get an instance of the Page with only the basic model methods so we simply extract
+    the ID of the page and hand it to the Page model imported directly.
+    """
+    Site = apps.get_model("wagtailcore", "Site")
+    site = Site.objects.filter(is_default_site=True).first()
+    if not site:
+        raise Exception(
+            "A default site is not set up. Please setup a default site before running this migration"
+        )
+    if not site.root_page:
+        raise Exception(
+            "No root (home) page set up. Please setup a root (home) page for the default site before running this migration"
+        )
+    return Page.objects.get(id=site.root_page.id)
+
+
+def remove_index_page(apps, schema_editor):
+    """
+    Remove the certificate index page
+    """
+    ContentType = apps.get_model("contenttypes.ContentType")
+    index_content_type, _ = ContentType.objects.get_or_create(
+        app_label="cms", model="certificateindexpage"
+    )
+    index_page = Page.objects.get(content_type_id=index_content_type.id)
+    if index_page:
+        index_page.delete()
+
+
+def create_index_page(apps, schema_editor):
```

Comment:
should It not be the part of `setup_index_pages` instead of adding in migrations ?

Reply:
Having it in the migration means the page will be set up automatically, so no need to run the command manually.